### ISSUE TITLE
gumbel-softmax 实现疑问

### ISSUE THREAD

**Issue by ownnaruto** [alipay/RGSL#2]
您好，关于模型中的gumbel_softmax函数实现，第53-55行的生成gumbel(0, 1)分布代码：
    gumbels = (
        -torch.empty_like(logits, memory_format=torch.legacy_contiguous_format).exponential_().log()
    )  # ~Gumbel(0,1)

这个地方的实现似乎并不是gumbel(0, 1)分布？
